api/AudioBufferSourceNode.json

```diff
@@ -365,24 +365,22 @@
               "version_added": false
             },
             "opera": {
-              "version_added": true,
-              "version_removed": "44"
+              "version_added": "17"
             },
             "opera_android": {
-              "version_added": true,
-              "version_removed": "43"
+              "version_added": "18"
             },
             "safari": {
-              "version_added": null
+              "version_added": true
```

Comment:
This was added in https://trac.webkit.org/changeset/150905/webkit, at WebKit 537.44.0. That would be Safari 7 and Safari for iOS 7. Can you test if that's right?

It was added for `OscillatorNode` at the same time. Can you update the data for api.OscillatorNode.onended to match what this ends up saying?

Reply:
Looks like it was actually added in Safari 6.1.  Updating the data now!

Reply:
Thanks!

Reply:
I checked Safari 6.1 was WebKit 537.43. That's very close. Perhaps these changes were backported to a release branch, or my method for determining the WebKit version at a change isn't entirely accurate. In any case it's best to keep verifying against the real releases when possible.